source/rust_verify/src/config.rs

```diff
@@ -465,6 +474,23 @@ pub fn parse_args_with_imports(
         "OPTION[=VALUE]",
     );
 
+    opts.optflag(
+        "",
+        OPT_COMPILE_WHEN_PRIMARY,
+        "When running under 'cargo verus', compile primary package(s)",
+    );
+    opts.optflag(
+        "",
+        OPT_COMPILE_WHEN_NOT_PRIMARY,
+        "When running under 'cargo verus', compile non-primary package(s)",
+    );
+    opts.optmulti(
+        "",
+        OPT_IMPORT_DEP_IF_PRESENT,
+        "When running under 'cargo verus', import .vir file for dependency",
+        "DEPENDENCY_NAME",
+    );
```

Comment:
I suspect these show up in `help`, right? Should they?

Reply:
How about something like this? https://github.com/verus-lang/verus/compare/cargo-integration...cargo-integration-internal-cargo-args (ignore the first of the two commits, I needed to cherry-pick it from `main` so that I could test things with my more up-to-date rustup)

Reply:
It would also be nice to move some of these strings to `tools/common/consts.rs` or similar, but we can take care of that later.